src/jaxsim/api/model.py

```diff
@@ -122,11 +129,15 @@ def build_from_model_description(
                 considered_joints=considered_joints
             )
 
+        terrain = terrain or JaxSimModel.__dataclass_fields__["terrain"].default
+        contact_model = contact_model or SoftContacts(terrain=terrain)
+
```

Comment:
Can this initialization be removed since `JaxSimModel.build` already handles the case in which `terrain=None` and `contact_model=None`?